src/biogeochem/CNFUNMod.F90

```diff
@@ -488,6 +487,8 @@ subroutine CNFUN(bounds,num_soilc, filter_soilc,num_soilp&
   integer   :: icost                             ! a local index
   integer   :: fixer                             ! 0 = non-fixer, 1
   ! =fixer 
+  !TODO, make namelist option
+  integer   :: nfix_method = 2                   ! 1 = Houlton, 2 = Bytnerowicz
```

Comment:
@slevis-lmwg tagging you here.

Reply:
@wwieder thanks. When I return to this in the morning, I'm tempted to update to the most recent ctsm tag, to avoid updating after the fact. Let me know if you have a reason to advise against that.

Reply:
I think that's a good idea

Reply:
Rethinking this idea because you suggested merging to b4b-dev. I will start by rebasing this to b4b-dev and go from there. Sorry if this sounds confusing. I will figure it out :-)